# clipss2 Highway 1.4.0 change notice
# Upstream: hwy/base.h; Copyright 2020 Google LLC; Apache-2.0.
# Diagnostic changes: guardiancrow, 2026-10-19; two stable highway file IDs.
# Notice implementation: 2026-10-20; replace one empty comment line.
# Upstream copyright/SPDX/license header and all existing line numbers are retained.

--- a/hwy/base.h
+++ b/hwy/base.h
@@ -1,6 +1,6 @@
 // Copyright 2020 Google LLC
 // SPDX-License-Identifier: Apache-2.0
-//
+// Modified by guardiancrow (2026-10-19) for clipss2: stable highway diagnostic file ID, 2 sites.
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
@@ -353,10 +353,10 @@
 #endif  // HWY_HEADER_ONLY
 
 #define HWY_WARN(format, ...) \
-  ::hwy::Warn(__FILE__, __LINE__, format, ##__VA_ARGS__)
+  ::hwy::Warn("highway", __LINE__, format, ##__VA_ARGS__)
 
 #define HWY_ABORT(format, ...) \
-  ::hwy::Abort(__FILE__, __LINE__, format, ##__VA_ARGS__)
+  ::hwy::Abort("highway", __LINE__, format, ##__VA_ARGS__)
 
 // Always enabled.
 #define HWY_ASSERT_M(condition, msg)               \
